src/client/components/UnreadMessages/CountUnreadMessages.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import './Unread.style.css';
+import { useParams } from 'react-router-dom';
+
+function CountUnreadMessages() {
+  const { userId, channelId } = useParams();
+
+  const [unreadMessages, setUnreadMessages] = useState([]);
+
+  useEffect(() => {
+    let mounted = true;
+    if (mounted) {
+      const unReadedMessages = async () => {
+        try {
+          const getUnreadMessages = await fetch(
+            `/api/unread?userId=${userId}&channelId=${channelId}`,
+          ).then((response) => response.json());
+
+          setUnreadMessages(getUnreadMessages);
+        } catch (erro) {
```

Comment:
Rename to `error`. Also, why are you returning it if you are not using it? Don't return it if you are not using it.